BDHelloPointSDk.xcodeproj/project.pbxproj

```diff
@@ -29,30 +28,18 @@
 		91F022CF226F252A002F2EFB /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 91F022CE226F252A002F2EFB /* main.m */; };
 /* End PBXBuildFile section */
 
-/* Begin PBXCopyFilesBuildPhase section */
-		919BEAD62276B5CF00EC9410 /* Embed Frameworks */ = {
-			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
-			dstPath = "";
-			dstSubfolderSpec = 10;
-			files = (
-				919BEAF222795B0B00EC9410 /* BDPointSDK.framework in Embed Frameworks */,
-			);
-			name = "Embed Frameworks";
-			runOnlyForDeploymentPostprocessing = 0;
-		};
-/* End PBXCopyFilesBuildPhase section */
-
 /* Begin PBXFileReference section */
 		78028E9A25DA47E7003A9BF4 /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9134A73827BCCEE900DA519C /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9134A73D27BCCF5C00DA519C /* BDPointSDK.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = BDPointSDK.framework; path = "Carthage/Checkouts/PointSDK-iOS/PointSDK/BDPointSDK.framework"; sourceTree = "<group>"; };
+		9160FED527BB2148001AEA99 /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9160FEDB27BC7C19001AEA99 /* BDPoint.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; path = BDPoint.xcframework; sourceTree = "<group>"; };
```

Comment:
I think this is a leftover from the previous method of manually adding xcframework?

Reply:
Yes, removed extra entries of xcframework